### PR TITLE
GA Install Astro on AWS Using a Cross-account Role

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1955 

@jwitz I wasn't sure what you had in mind for this content - a standalone topic, or incorporating it into the existing AWS topic. Feel free to make any changes you think are necessary.

### CODE REVIEW

**Review by jwitz — CHANGES_REQUESTED** [review]
@chomipi88 My understanding of this project was that this setup would replace our existing AWS installation steps, is that correct? If so, @kwcanuck can you:

- Delete this doc 
- Set a redirect to the current install doc
- Merge this content with the existing AWS content.

**Comment by chomipi88** [comment]
@jwitz yes that's right

File: astro/install-aws.md [hunk 1]
```diff
@@ -71,30 +58,29 @@ Astronomer support will create a cluster within your AWS account that hosts the
     - `https://updates.astronomer.io/`
     - `https://install.astronomer.io/`
 
-Astro requires a clean AWS account with a minimum set of EC2 service quotas. For security reasons, the install process is not currently supported on an AWS account that has other tooling running in it. For instructions on creating a new AWS account, follow [AWS documentation](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
+Astro requires a clean AWS account with the minimum EC2 service quotas. For security reasons, the install process is not currently supported on an AWS account that has other tooling running in it. For instructions on creating a new AWS account, see [How do I create and activate a new AWS account?](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
```
Comment (chomipi88) [inline astro/install-aws.md:61]:
```suggestion
Astro requires a dedicated AWS account with the minimum EC2 service quotas. For security reasons, the install process is not currently supported on an AWS account that has other resources running in it. For instructions on creating a new AWS account, see [How do I create and activate a new AWS account?](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
```

File: astro/install-aws.md [hunk 2]
```diff
@@ -117,110 +103,65 @@ If you want to continue with the second option, you'll additionally need:
...
 
-<Tabs
-    defaultValue="managementconsole"
-    groupId= "step-3-create-a-cross-account-iam-role-for-astro"
-    values={[
-        {label: 'AWS Management Console', value: 'managementconsole'},
-        {label: 'AWS Command Line', value: 'commandline'},
-    ]}>
-<TabItem value="managementconsole">
+1. Log in to your AWS CloudFormation instance.
```
Comment (chomipi88) [inline astro/install-aws.md:118]:
```suggestion
1. Log in to your [AWS CloudFormation console](https://console.aws.amazon.com/cloudformation).
```

File: astro/install-aws.md [hunk 3]
```diff
@@ -117,110 +103,65 @@ If you want to continue with the second option, you'll additionally need:
...
-                ],
-            },
-        },
-    }
-    ```
+When new features or functionality are added to Astro, Astronomer might need to modify cross-account role permissions. When setting permissions, Astronomer adheres to the least-privilege permissions standard and adds only the permissions necessary to use the new feature or functionality.
 
-</TabItem>
-</Tabs>
+Astronomer support notifies your Organization when any changes are made to the policies that expand user access. Notifications will include an explanation of the changes being made and why the change was necessary.
```
Comment (chomipi88) [inline astro/install-aws.md:132]:
```suggestion
Astronomer support notifies your Organization when any changes are made to the policies that expand cross-account role access. Notifications will include an explanation of the changes being made and why the change was necessary. 
```

**Comment by chomipi88** [comment]
LGTM

**Comment by jwitz** [comment]
@chomipi88 I'll be able to review this in about an hour. Do we need to also add a release note for the March 1 release? Seems like a big enough deal to notify on there.

File: astro/install-aws.md [hunk 4]
```diff
@@ -39,29 +39,16 @@ To install Astro in a dedicated AWS account owned by your organization, you'll c
 
 - Create an account on Astro.
 - Share AWS account information with Astronomer support.
-- Create a cross-account IAM role that Astro can assume within your new AWS account.
+- Create the IAM policies used by Astro. This includes a cross-account IAM role that Astro can assume and permissions boundaries for the Astro operational roles. For more information about permissions boundaries, see [Permissions boundaries for IAM entities](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html).
```
Comment (jwitz) [inline astro/install-aws.md:42]:
```suggestion
- Create the IAM policies used by Astro. This includes a cross-account IAM role that Astro can assume and [permissions boundaries](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html) for the Astro operational roles.
```
For v optional reference content, I prefer if we save real estate by hyperlinking to the word being defined. Don't feel strongly either way tho.

File: astro/install-aws.md [hunk 5]
```diff
@@ -71,30 +58,29 @@ Astronomer support will create a cluster within your AWS account that hosts the
...
 
-If you have one or more existing AWS accounts, you can use [AWS Organizations](https://aws.amazon.com/organizations/) to manage billing, users, and more in a central place. For more information on how to add your Astro AWS account to your AWS Organization, read [Amazon's documentation](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_invites.html).
+If you have one or more existing AWS accounts, you can use [AWS Organizations](https://aws.amazon.com/organizations/) to manage billing, users, and more in a central place. For more information on how to add your Astro AWS account to your AWS Organization, see [Inviting an AWS account to join your organization](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_invites.html).
+
 :::
 
 #### VPC peering prerequisites (Optional)
 
-If any AWS resources are on a private network, you can choose between two options:
+The following options are available when your AWS resources are on a private network:
```
Comment (jwitz) [inline astro/install-aws.md:80]:
```suggestion
The following options are available to connect Astro to AWS resources on a private network:
```

File: astro/install-aws.md [hunk 6]
```diff
@@ -71,30 +58,29 @@ Astronomer support will create a cluster within your AWS account that hosts the
...
 
 - Allow traffic through the public internet and use allow-lists for communication.
```
Comment (jwitz) [inline astro/install-aws.md:82]:
```suggestion
- Allow traffic through the public internet and use allowlists for communication.
```

File: astro/install-aws.md [hunk 7]
```diff
@@ -117,110 +103,65 @@ If you want to continue with the second option, you'll additionally need:
...
-                    "Principal": { "AWS": "arn:aws:iam::406882777402:root" },
-                    "Action": "sts:AssumeRole",
-                    "Condition": { "StringEquals": { "sts:ExternalId": "" } },
-                  },
-                ],
-            },
-        },
-    }
-    ```
+When new features or functionality are added to Astro, Astronomer might need to modify cross-account role permissions. When setting permissions, Astronomer adheres to the least-privilege permissions standard and adds only the permissions necessary to use the new feature or functionality.
```
Comment (jwitz) [inline astro/install-aws.md:130]:
```suggestion
When new features or functionality are added to Astro, Astronomer might need to modify cross-account role permissions. When setting permissions, Astronomer adheres to the least-privilege permissions standard and adds only the permissions necessary for new features or functionality.
```

File: astro/install-aws.md [hunk 8]
```diff
@@ -117,110 +103,65 @@ If you want to continue with the second option, you'll additionally need:
...
 
-</TabItem>
-</Tabs>
+Astronomer support notifies your Organization when any changes are made to the policies that expand cross-account role access. Notifications will include an explanation of the changes being made and why the change was necessary. 
+
+Astronomer can reduce the access available to the policies without notification.
+
+#### Monitor the policies for changes (optional)
+
+You can use CloudTrail to monitor changes to Astro policies. Access to CloudTrail has been limited to prevent the accidental modification or deletion of CloudTrail logs by Astronomer support. The following table lists the events that you should monitor.
```
Comment (jwitz) [inline astro/install-aws.md:138]:
```suggestion
You can use CloudTrail to monitor changes to Astro policies. The following table lists the events that you should monitor. 
```
Not sure what this sentence is doing here, let's remove it and I'll add it back once I figure out when it was added and why?

**Review by jwitz — APPROVED** [review]
LGTM, I also added a release note that you are free to reword @kwcanuck !

File: astro/install-aws.md (hunk 8 above)
Comment (chomipi88) [reply astro/install-aws.md:138]: Let's leave it. This is actually an important detail.

**Pull request merged**